apps/mailroom-nest/src/app/app.controller.ts

```diff
@@ -1,18 +1,37 @@
-import { Body, Controller, Get, Post, UploadedFiles, UseFilters, UseInterceptors } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Post, UploadedFiles, UseFilters, UseInterceptors } from '@nestjs/common';
 import { AnyFilesInterceptor } from '@nestjs/platform-express';
 
 import 'multer';
 
 import { IMailroomEmailOutbound, LogToDatabaseInterceptor, MailroomExceptionFilter } from '@tamu-gisc/mailroom/common';
 import { Mailer } from '@tamu-gisc/oidc/common';
 
+import { AppService } from './app.service';
+
 @Controller()
 export class AppController {
+  constructor(public readonly service: AppService) {}
+
   @Get()
   public isRunning() {
     return 200;
   }
 
+  @Get('all')
+  public getAllEmails() {
+    return this.service.getAllEmails();
+  }
+
+  @Get(':id')
+  public getEmail(@Param('id') id) {
+    return this.service.getEmail(id);
+  }
+
+  @Delete(':id')
+  public deleteEmail(@Param('id') id) {
+    return this.service.deleteEmail(id);
+  }
+

```

Comment:
I should have caught this before, but apps should never contain logic however small they are.

This controller and accompanying service should be in a lib.

Reply:
Made a new PR for the boilerplate #266, rebased `mailroom-angular-alpha` onto that, and implemented everything in `mailroom-data-api`.

Reply:
Referring to #266, I don't see these routes implemented there.

Reply:
Fixed.